egs/rm/s5/local/chain/run_tdnn_5o.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# this script is a modified version of run_tdnn_5n.sh. It uses
+# a new configs convention for chain model after kaldi 5.2.
```

Comment:
can you please mention that the results are in the RESULTS file?  People normally look for them here.
Or duplicate them here.

Reply:
.. and also, please move all these tuning scripts to a subdirectory tuning/, and make a link run_tdnn.sh -> tuning/run_tdnn_5o.sh.

Reply:
Thank you, I moved scripts and added mention in RESULTS file

Reply:
It doesn't look like you removed the scripts from their original location.  Also make sure the soft link run_tdnn.sh is there; it seems to be missing (you'll have to "git add" it).

Reply:
I'm sorry, I missed. Thank you.